### PR TITLE
Update Bindings: v2.0.9

### PR DESCRIPTION
**Pull request by jackgibson1**
Update bindings to `v2.0.9`.
API version `v1.355.2`. 

The previous version (`v2.0.8`) was completely fine when used independently. However the `architecture` enum field on the `CranPackageUploadRequest` class had an incompatibility with our [CLI here](https://github.com/cloudsmith-io/cloudsmith-cli/blob/master/cloudsmith_cli/core/api/packages.py#L251) so had to be yanked to avoid CLI breakages.

### CODE REVIEW

**Review by dgonzalez — APPROVED** [review]

File: bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb [hunk 1]
```diff
@@ -35,10 +35,6 @@
   describe 'test attribute "architecture"' do
     it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
-      # validator = Petstore::EnumTest::EnumAttributeValidator.new('String', ["arm64", "x86_64"])
-      # validator.allowable_values.each do |value|
-      #   expect { @instance.architecture = value }.not_to raise_error
-      # end
```
Comment (nicholasmccrea) [inline bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb:41]: Is this test needed anymore? 🤔

**Review by nicholasmccrea — APPROVED** [review]
LGTM

File: bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb (hunk 1 above)
Comment (jackgibson1) [reply bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb:41]: Nope, the architecture field is no longer an enum/choice field (due to the incompatibility with the CLI), so the allowable values isn't necessary now.

Comment (nicholasmccrea) [reply bindings/ruby/src/spec/models/cran_package_upload_request_spec.rb:41]: Cool cool, worth just deleting then?

**Pull request merged**